content/docs/tech-insights/builtin-data-sources/index.md

```diff
@@ -10,21 +10,219 @@ description: Built-in Data Sources.
 
 Built-in Data Sources use integration configurations from common Backstage plugins and annotations provided by them. You can see configuration instructions for integrations on their respective documentation pages in [the Plugins & Integrations page](/docs/integrations/).
```

Comment:
We should mention here somewhere that users can make their own data sources if they need something we haven't provided out of the box. We should also link to the instructions for doing that.